### PR TITLE
update readme and add home

### PR DESCRIPTION
**Pull request by YunjieLi**
1. Update readme.md to include instructions for updating layout
2. Add home/house icon ([#281](https://github.com/mapbox/maki/issues/281)).
3. Add Buddhist icon

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,21 +0,0 @@
-Changelog
```
Comment (samanpwbb) [inline CHANGELOG.md]: Looks like this was an accidental change.

**Comment by samanpwbb** [comment]
The basic form looks good. Some of your nodes are not quite pixel aligned. I recommend making sure you have a pixel grid set up in illustrator that places a gridline at every pixel (or even every half pixel), and make sure snap to grid `Ctrl + Shft + '` is turned on.

File: CHANGELOG.md (hunk 1 above)
Comment (YunjieLi) [inline CHANGELOG.md]: That's because I reverted a version I think

**Comment by YunjieLi** [comment]
Thanks for the feedback! @samanpwbb 
I'm not so sure what do you mean by pixel aligned? I have the artboard setup like this. Does the rooftop triangle looks not aligned?
image

**Comment by samanpwbb** [comment]
Make sure pixel snapping is turned on: vertex positions should be at integer values whenever possible, to ensure the crispest rendering possible: 
screen shot 2016-08-22 at 7 31 50 pm

**Comment by YunjieLi** [comment]
If we'll have to bend the guidelines anyway, can we do this @natslaughter 

image
buddhism-11

image

**Comment by aparlato** [comment]
cc @natslaughter @samanpwbb

**Comment by natslaughter** [comment]
@aparlato 
Is there any way the home icons could avoid having geometry in the 'trim area?' The 11-pixel icon is pushing to the edges, which might cause some tight, odd conditions when a background or frame is applied to the icon.

**Pull request merged**